src/main/java/com/fabriccommunity/spookytime/common/SpookyBlocks.java

```diff
@@ -0,0 +1,59 @@
+package com.fabriccommunity.spookytime.common;
+
+import com.fabriccommunity.spookytime.SpookyTime;
+import com.fabriccommunity.spookytime.block.CarvablePumpkinBlock;
+import com.fabriccommunity.spookytime.block.DeceasedGrassBlock;
+import com.fabriccommunity.spookytime.block.TinyPumpkinBlock;
+import com.fabriccommunity.spookytime.block.entity.CarvablePumpkinBlockEntity;
+import net.fabricmc.fabric.api.block.FabricBlockSettings;
+import net.minecraft.block.*;
+import net.minecraft.block.entity.BlockEntityType;
+import net.minecraft.item.BlockItem;
+import net.minecraft.item.Item;
+import net.minecraft.item.ItemGroup;
+import net.minecraft.sound.BlockSoundGroup;
+import net.minecraft.util.Identifier;
+import net.minecraft.util.registry.Registry;
+
+public class SpookyBlocks {
+    public static BlockEntityType<CarvablePumpkinBlockEntity> CARVABLE_PUMPKIN_BLOCK_ENTITY;
```

Comment:
Block Entities should be done in `SpookyBlockEntities`, not `SpookyBlocks`.

Reply:
Where's SpookyBlockEntities?
I don't see it in com.fabriccommunity.spookytime.block.entity

Reply:
It's in the `registry` package, along with all the other registry classes.

Reply:
thanks